From 93186220527e258496c6ba49df30faf365534ee2 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Mon, 27 Jun 2022 10:10:55 -0700
Subject: [PATCH] Remove wtf if no display nits mapping exists

Devices without a backlight will not provide nits mappings via hardware
composer, which is perfectly normal and has no side effects.

Bug: 234889959
Change-Id: I7f5f0e4e24bfb38335dfe09102b36b106a904d2d
(cherry picked from commit 93186220527e258496c6ba49df30faf365534ee2)
---
 .../java/com/android/server/display/DisplayDeviceConfig.java     | 1 -
 1 file changed, 1 deletion(-)

diff --git a/services/core/java/com/android/server/display/DisplayDeviceConfig.java b/services/core/java/com/android/server/display/DisplayDeviceConfig.java
index 255494f986673..1d0acfad3be6f 100644
--- a/services/core/java/com/android/server/display/DisplayDeviceConfig.java
+++ b/services/core/java/com/android/server/display/DisplayDeviceConfig.java
@@ -246,7 +246,6 @@ public float getBacklightFromBrightness(float brightness) {
      */
     public float getNitsFromBacklight(float backlight) {
         if (mBacklightToNitsSpline == null) {
-            Slog.wtf(TAG, "requesting nits when no mapping exists.");
             return NITS_INVALID;
         }
         backlight = Math.max(backlight, mBacklightMinimum);
